src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/Internal/ManifestGenerator.cs

```diff
@@ -9,15 +9,17 @@
 using Microsoft.AspNetCore.Mvc.ApplicationParts;
 using Microsoft.AspNetCore.Mvc.Razor.Compilation;
 using Microsoft.AspNetCore.Mvc.Razor.Internal;
+using Microsoft.AspNetCore.Mvc.RazorPages.ApplicationFeature;
+using Microsoft.AspNetCore.Mvc.RazorPages.Infrastructure;
```

Comment:
The currently manifest lives in this. I'll do a follow up to switch to attributes which are in the default assembly namespace.